tests/h/models/groups_test.py

```diff
@@ -155,15 +147,57 @@ def test_documents_when_group_has_no_documents(group):
     assert group.documents() == []
 
 
-def test_acl(group, factories):
-    group.pubid = 'testing-pubid'
-    group.creator = factories.User(username='luke', authority='foobar.org')
+class TestGroupACL(object):
+    def test_authority_joinable(self, group):
+        group.joinable_by = JoinableBy.authority
+
+        assert (security.Allow, 'authority:example.com', 'join') in group.__acl__()
```

Comment:
Ah yes, much safer. I was loosely basing these tests on what we do for the annotation model. But they always check the full list of permissions, which doesn't have the same issue.
Fixed